doc/sphinx/addendum/type-classes.rst

```diff
@@ -388,6 +388,16 @@ Summary of the commands
       equivalent to ``Hint Resolve ident : typeclass_instances``, except it
       registers instances for :cmd:`Print Instances`.
 
+   .. flag:: Instance Generalized Output
+
+      .. deprecated:: 8.13
+
+      Disabled by default, this provides compatibility with |Coq|
+      version 8.12 and earlier.
+
+      When enabled, the type of the instance is implicitly generalized
+      over unbound variables as though surrounded by ``\`{}``.
```

Comment:
```suggestion
      over unbound and :ref:`generalizable <implicit-generalization>` variables as though surrounded by ``\`{}``.
```

I think it is important to add a reference to the implicit generalization section but it doesn't have to be like in my suggestion.